ledger-tool/src/ledger_utils.rs

```diff
@@ -362,10 +362,8 @@ pub fn load_and_process_ledger(
             let tss_blockstore = if enable_rpc_transaction_history {
                 Arc::new(open_blockstore(
                     blockstore.ledger_path(),
+                    arg_matches,
                     AccessType::PrimaryForMaintenance,
-                    None,
-                    false,
-                    false,
```

Comment:
This could be a possible behavior change, couldn't it? I assume we do want to respect those flags in the `enable_rpc_transaction_history` case, but just pointing it out.

Reply:
> This could be a possible behavior change, couldn't it? 

It should not. By this point, a `Blockstore` has already been opened and passed into `load_and_process_ledger()`. So, all of these flags will have already been parsed and used to open a `Blockstore` successfully if we make it to this point.